src/main/java/net/minecraftforge/fml/common/ILanguageAdapter.java

```diff
@@ -166,7 +176,8 @@ public void setInternalProxies(ModContainer mod, Side side, ClassLoader loader)
 
                             setProxy(target, proxyTarget, proxy);
                         }
-                        catch (Exception e) {
+                        catch (Exception e)
+                        {
```

Comment:
Will do so, I assumed it was okay since the old formatting didn't match the standards, but I understand if you want me to leave it as is.